src/http/client.ts

```diff
@@ -47,10 +50,12 @@ export function createHttpClient({
     );
   }
 
-  http.interceptors.response.use(
-    camelizeResponseData,
-    typeReceivedErrorsInterceptor
-  );
+  http.interceptors.response.use(null, retryInterceptor(http));
```

Comment:
Could be we don't want these interceptors?

```js
  http.interceptors.response.use(null, retryInterceptor(http));
  http.interceptors.response.use(null, typeReceivedErrorsInterceptor);
  ```

Reply:
The `typeReceivedErrorsInterceptor` was already there, so I would keep that one.

Reply:
I don't think the `retryInterceptor` can hurt here?  If the user sets the amount of retries in the config, he will expect it to happen, also if the call is not authenticated.